src/Appwrite/Utopia/Response/Model/Bucket.php

```diff
@@ -66,6 +66,13 @@ public function __construct()
                 'example' => ['jpg', 'png'],
                 'array' => true
             ])
+            ->addRule('compression', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Compression method chosen.',
```

Comment:
Let's list the options with links to wikipedia.

Reply:
In the description in both places?

Reply:
done